octoprint_FriendlyBeeper/__init__.py

```diff
@@ -13,12 +13,15 @@ class FriendlybeeperPlugin(octoprint.plugin.StartupPlugin,
                            octoprint.plugin.AssetPlugin,
                            octoprint.plugin.SimpleApiPlugin):
 
-    def do_beep(self, event):
+    def do_beep(self, event, settings = None):
         now = datetime.now()
 
+        if settings == None:
+            settings = self._settings.get_all_data();
```

Comment:
would it make sense to just do
```python
def do_beep(self, event, settings = self._settings.get_all_data()):
```
instead? Or was `settings = None` done just to keep the line shorter?

Reply:
I find it easier to read that way, but it's my personal preference. I'll change that.

Reply:
I changed it back, as python evaluates the default argument at definition time, not at call-time. See this example:

```
 
def f1():
    print("f1")
    return "default argument"

def f2(str = f1()):
    print("f2")
    print(str)


print("start")
f2()
```
Output:
```
f1
start
f2
default argument
```